source/ocsp-support/ocsp-support.rst

```diff
@@ -156,6 +156,16 @@ per MongoClient basis.
    ``tlsAllowInvalidCertificates=true`` are specified alongside the
    option to enable certificate revocation checking.
 
+TLS Requirements
+----------------
+`Server Name Indication
+<https://en.wikipedia.org/wiki/Server_Name_Indication>`__ (SNI) MUST BE
```

Comment:
IIRC, RST link syntax only calls for a single underscore.

Reply:
I think the double underscore is acceptable for one off links? (at least according to https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use)

Reply:
For the other double underscore, the link was broken because we have two sections with the same name (but at different levels) so I've changed it to use a direct reference

Reply:
Ah, my mistake. I wasn't aware that a single underscore for URLs created a reference for future use. Feel free to ignore this.